runconfig/parse_unix.go

```diff
@@ -7,51 +7,53 @@ import (
 	"strings"
 )
 
-func parseNetMode(netMode string) (NetworkMode, error) {
-	parts := strings.Split(netMode, ":")
+// ValidateNetMode ensures that the various combinations of requested
+// network settings are valid.
+func ValidateNetMode(c *Config, hc *HostConfig) error {
+	// We may not be passed a host config, such as in the case of docker commit
+	if hc == nil {
+		return nil
+	}
```

Comment:
I'm still not sure I agree here. I have (some time back now) done downlevel CLI checking and didn't hit a problem that indicated any backwards compatibility issues. Are you simply asking that I include "" in the check for possible valid cases here?